.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+# NOTE(jonesbr): Copied from https://github.ibm.com/Whitewater/whitewater-detect-secrets/blob/master/user-config/.pre-commit-config.yaml
```

Comment:
This note won't help folks outside IBM.  Are there any alternatives at https://github.com/ibm/detect-secrets which I believe is public?

Reply:
Updated to point to the public repo